pgmpy/tests/test_readwrite/test_XMLBIF.py

```diff
@@ -2,7 +2,17 @@
 import unittest
 import numpy as np
 import numpy.testing as np_test
-from pgmpy.readwrite import XMLBIFReader
+from pgmpy.readwrite import XMLBIFReader, XMLBIFWriter
+try:
+    from lxml import etree
+except ImportError:
+    try:
+        import xml.etree.cElementTree as etree
+    except ImportError:
+        try:
+            import xml.etree.ElementTree as etree
+        except ImportError:
+            warnings.warn("Failed to import ElementTree from any known place")
```

Comment:
@jaidevd i can't find any better way. Stackoverflow shows the use of nested try except only.http://stackoverflow.com/questions/342437/best-way-to-import-version-specific-python-modules